lmdeploy/cli/utils.py

```diff
@@ -241,7 +241,7 @@ def log_level(parser):
         import logging
         return parser.add_argument('--log-level',
                                    type=str,
-                                   default='ERROR',
+                                   default='WARNING',
```

Comment:
Change this would cause warning for each request:
```
total sequence length (103 + 65441) exceeds `session_len` (65544), `request_output_len` is truncated to 65440
```
Maybe we have to further minus 1 for max_new_tokens in https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/serve/async_engine.py#L580-L591

Reply:
Ref https://github.com/InternLM/lmdeploy/blob/main/src/turbomind/models/llama/LlamaBatch.cc#L349-L352

Reply:
Maybe we could make this change in a separate PR. I will change the info level back to `ERROR` first.

Reply:
> Maybe we could make this change in a separate PR. I will change the info level back to `ERROR` first.

That looks good to me.

Reply:
> Maybe we could make this change in a separate PR. I will change the info level back to `ERROR` first.

done cc @AllentDan